kafka/consumer.c

```diff
@@ -460,7 +332,9 @@ lua_consumer_poll_rebalances(struct lua_State *L) {
         luaL_error(L, "Usage: count, err = consumer:poll_rebalances(limit)");
 
     consumer_t *consumer = lua_check_consumer(L, 1);
-    if (consumer->event_queues == NULL || consumer->event_queues->rebalance_queue == NULL || consumer->event_queues->rebalance_cb_ref == LUA_REFNIL) {
+    if (consumer->event_queues == NULL ||
```

Comment:
Is it possible to use LUA_RDKAFKA_POLL_FUNC here?

Reply:
This callback is too custom. No it's impossible.